source/_data/data.json

```diff
@@ -77,11 +77,11 @@
   "plainText": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quis, unde, dolore? Ex fugit, necessitatibus tempore neque ullam nostrum ducimus alias beatae voluptatibus iusto, perspiciatis tenetur consectetur reprehenderit, harum quae rerum! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Suscipit corrupti fuga, quas quisquam perspiciatis. Consequatur, atque ducimus dolores natus officia qui rerum consectetur repellat ullam accusantium laboriosam, reprehenderit nemo deleniti",
   "htmlText": "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quis, unde, dolore? Ex fugit, necessitatibus tempore neque ullam nostrum ducimus alias beatae voluptatibus iusto, perspiciatis tenetur consectetur reprehenderit, harum quae rerum!</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Suscipit corrupti fuga, quas quisquam perspiciatis. Consequatur, atque ducimus dolores natus officia qui rerum consectetur repellat ullam accusantium laboriosam, reprehenderit nemo deleniti</p>",
   "footerLegal": "© 2017 - FTI Touristik GmbH. Alle Rechte vorbehalten. (used atom: paragraph)",
-  "footerMenu": [
-    {"footerMenuItem": "Kontakt"},
-    {"footerMenuItem": "Unternehmen"},
-    {"footerMenuItem": "Impressum"},
-    {"footerMenuItem": "AGBs"}
+  "extraMenu": [
```

Comment:
Good idea.
There will be a cleanup for the json files in the future. We just wanted to keep the numbers of files low. 😉Also, we want to make sure that the data is not used. Since this takes time and this needs to go out soon, we want to postpone it a little.